bors.toml

```diff
@@ -1,7 +1,4 @@
 status = [
-  # "ci 1.8.5 - ubuntu-latest",
```

Comment:
It looks like CI is working on ubuntu and macos. Let's uncomment this and update 1.8.5 to 1.9

Reply:
Buildkite is a better success metric, but we do have some tests in test/runtests.jl that are not run on buildkite, so we really should be checking that.